earth_enterprise/src/fusion/mtjtools/scroll/zoom.h

```diff
@@ -545,7 +545,7 @@ class Image
     void *item = ::malloc(size);
 
     if (log != nullptr)
-      fprintf(log, "A 0x%08x %8d\n", item, size);
+      fprintf(log, "A 0x%08x %8lu\n", static_cast<unsigned int>(item), static_cast<unsigned long>(size));
```

Comment:
Minor: there is a `%p` printf specifier for `void *` that would make more sense for `item`. The same applies to the two changes below.